### PR TITLE
Prepare noetic release

### PR DESCRIPTION
**Pull request by JeroenDM**
TODO for noetic version:

- [x] Remove opw_kinematics submodule and add it as a released dependency. (Figure out how to correctly add it in CMakeList.txt.)
- [x] Check if tests run on noetic (figure out the problem with the missing `industrial_robot_client` package on noetic to make `kuka_test_resources` work) (**Solution**: I build industrial_robot_client from source).
- [x] Run prerelease tests.
- [x] Update changelog
- [x] Add to rosdistro index

[Link](http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease) to release doc.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -13,6 +13,7 @@ find_package(catkin REQUIRED COMPONENTS
   moveit_core
   roscpp
   pluginlib
+  opw_kinematics
```
Comment (gavanderhoorn) [inline CMakeLists.txt]: `opw_kinematics` is not a Catkin package. It's a plain CMake package, so you'd need to do something like:

```cmake
find_package(opw_kinematics REQUIRED)

...

target_link_libraries(${MOVEIT_LIB_NAME}
  ${catkin_LIBRARIES}
  opw_kinematics::opw_kinematics
)
```

Comment (JeroenDM) [reply CMakeLists.txt]: Yep indeed, thanks for the pointer! I was not sure about the namespace thing `opw_kinematics::opw_kinematics`.

**Comment by gavanderhoorn** [comment]
Ping? :)

**Comment by JeroenDM** [comment]
@gavanderhoorn Better late than never...sorry. https://github.com/ros/rosdistro/pull/30842

**Comment by gavanderhoorn** [comment]
thanks for the release.

**Pull request merged**